views/partials/_example.njk

```diff
@@ -1,5 +1,11 @@
+
 {% macro example(params) %}
-{% set examplePath = "src/" + params.group + "/" + params.item + "/" + params.example + "/index.njk" %}
+{% set exampleRoot = "src/" + params.group + "/" + params.item + "/" + params.example + "/" %}
+{% if params.customCode %}
```

Comment:
This is just to avoid having robots.txt in the code that's copied right? I wonder if there's a better way of doing it, as we also need to maintain two 99% similar copies of the code. Could we regex it out somehow?